velox/expression/CastExpr.h

```diff
@@ -113,6 +113,13 @@ class CastExpr : public SpecialForm {
       const TypePtr& toType,
       VectorPtr& result);
 
+  VectorPtr applyUnknown(
```

Comment:
This method is not using any member variables, hence, can be a free function in .cpp file. Also, the implementation is a one-liner and there is only one caller, hence, this function is not needed at all.